Sources/TouchDrawView.swift

```diff
@@ -221,6 +221,15 @@ open class TouchDrawView: UIView {
 
 extension TouchDrawView {
 
+    open override func gestureRecognizerShouldBegin(_ gestureRecognizer: UIGestureRecognizer) -> Bool {
+        // This prevents the drop down to dismiss gesture in modal presentations from iOS 13 on.
+        // See also: https://stackoverflow.com/questions/56718552/disable-gesture-to-pull-down-form-page-sheet-modal-presentation
+        if gestureRecognizer is UIPanGestureRecognizer {
```

Comment:
You could just `return !(gestureRecognizer is UIPanGestureRecognizer)`

Reply:
Sure. This is just personal preference by me, to make it super easy to understand. I try to prevent `!` negations, because you can overlook it so easily...
But you can change it, if you want!